app/controllers/catalog_controller.rb

```diff
@@ -203,8 +203,8 @@ class CatalogController < ApplicationController
     config.add_facet_fields_to_solr_request! # deprecated in newer Blacklights
 
     config.add_search_field 'text', label: 'All Fields'
+    config.add_sort_field 'score desc', label: 'Relevance', default: true
```

Comment:
Yay!  We can turn on relevancy ranked search results!

I probably didn't have to move the line, but ... it works and who cares?